src/CommunityToolkit.Maui.Core/Layouts/UniformItemsLayoutManager.shared.cs

```diff
@@ -86,7 +86,7 @@ public override Size Measure(double widthConstraint, double heightConstraint)
 	int GetColumnsCount(int visibleChildrenCount, double widthConstraint)
 	{
 		var columnsCount = visibleChildrenCount;
-		if (childWidth != 0 && !double.IsPositiveInfinity(widthConstraint))
+		if (visibleChildrenCount > 0 && childWidth != 0 && !double.IsPositiveInfinity(widthConstraint))
 		{
 			columnsCount = Math.Clamp((int)(widthConstraint / childWidth), 1, visibleChildrenCount);
```

Comment:
The problem seems to be here, when visibleColumn = 0 and 1(min)>0(max)

Reply:
OK, Now,Forced return 1

Reply:
I don't know what the preferred option is but I believe the two suggestions I have added would also solve the issue

Reply:
It won't let me make the suggestion on line 99 but I would also suggest we change:

```csharp
return Math.Min(columnsCount, uniformItemsLayout.MaxColumns);
```

to

```csharp
return Math.Clamp(columnsCount, 1, uniformItemsLayout.MaxColumns);
```